src/sentry/static/sentry/app/views/performance/trends/chart.tsx

```diff
@@ -55,6 +61,38 @@ function transformEventStats(data: EventsStatsData, seriesName?: string): Series
   ];
 }
 
+function transformEventStatsSmoothed(data: Series[], seriesName?: string): Series[] {
+  const currentData = data[0].data;
+  const resultData = [] as SeriesDataUnit[];
```

Comment:
```suggestion
  const resultData: SeriesDataUnit[] = [];
```
Should also work and doesn't use a cast operator.